docs/docs/contactus.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Contact Us
+---
+
+
+# Contact Us
+
+##Found a bug or have a great feature idea?
+Use Issues https://github.com/salesforce/refocus/issues. 
```

Comment:
square brackets around the word "Issues" then no space then parentheses around the url